src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/FilterOptionMeta.cs

```diff
@@ -2,11 +2,13 @@ namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
 
 public class FilterOptionMeta
 {
-    public required string PublicId { get; set; }
-
-    public required int PrivateId { get; set; }
+    public int Id { get; set; }
```

Comment:
`Guid`

Reply:
**Discussed:**

Only using `Guids` for publicly available IDs. Using `int` for strictly internally available IDs as it is more performant.